substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/DebugInfoBase.java

```diff
@@ -236,40 +236,38 @@ public void installDebugInfo(DebugInfoProvider debugInfoProvider) {
              */
             String fileName = debugCodeInfo.fileName();
             Path filePath = debugCodeInfo.filePath();
-            Path cachePath = debugCodeInfo.cachePath();
-            String className = TypeEntry.canonicalize(debugCodeInfo.className());
-            String methodName = debugCodeInfo.methodName();
+            String className = TypeEntry.canonicalize(debugCodeInfo.ownerType());
+            String methodName = debugCodeInfo.name();
             String symbolName = debugCodeInfo.symbolNameForMethod();
             String paramSignature = debugCodeInfo.paramSignature();
-            String returnTypeName = TypeEntry.canonicalize(debugCodeInfo.returnTypeName());
+            String returnTypeName = TypeEntry.canonicalize(debugCodeInfo.valueType());
             int lo = debugCodeInfo.addressLo();
             int hi = debugCodeInfo.addressHi();
             int primaryLine = debugCodeInfo.line();
             boolean isDeoptTarget = debugCodeInfo.isDeoptTarget();
```

Comment:
I think we probably ought to migrate DebugCodeInfo::isDeoptTarget() to DebugMethodInfo and cache this value in the MethodEntry when we create it. The test simply checks that the hosted method name ends in "**" and coudl be done up front when processing the DebugMethodInfo. If we do that then there is no need to retrieve isDeoptTarget here and pass it into makePrimaryRange. The latter function can read it off the methodEntry.